src/helpers.js

```diff
@@ -17,7 +17,7 @@ function calculateTextAlignment(size, options) {
   const textAlign = options.textAlign;
   if (textAlign === 'center') {
     return x + width / 2;
-  } else if (textAlign === 'end' || textAlign === 'right') {
+  } else if (textAlign === 'end') {
```

Comment:
Why this? Types still allow left/right too?

Reply:
reapplied. I have removed because I thought only `start` and `end` were accepted.